CMakeLists.txt

```diff
@@ -172,6 +176,15 @@ if(UNIX AND NOT APPLE)
   endif()
   find_package(X11 COMPONENTS X11_xcb)
 endif()
+
+find_package(Qt6 REQUIRED COMPONENTS Protobuf)
+if(NOT Protobuf_FOUND)
+  find_package(Protobuf REQUIRED)
+endif()
+if(NOT TARGET protobuf::protoc)
+  message(FATAL_ERROR "Missing Protobuf compiler.")
```

Comment:
Remove: No need to use Protobuf directly.

Reply:
See above